db/schema.rb

```diff
@@ -137,7 +137,7 @@
     t.integer "age_range_min"
     t.integer "age_range_max"
     t.text "subjects", default: [], null: false, array: true
-    t.datetime "recommended_at", precision: nil
+    t.datetime "recommended_at"
```

Comment:
Looks like there are some unintended changes in this file, not entirely sure where they're coming from though. 🤔

Reply:
So clearly this is somethign to do with some configuration on my computer when i ran the migration; Im not sure how to undo the removals of precision: nil without rolling back my changes. Any idea how i can get my schema to be consistent with main?